examples/transform/src/main/java/org/apache/apex/examples/transform/SimpleTransformApplication.java

```diff
@@ -19,43 +19,72 @@
 
 package org.apache.apex.examples.transform;
 
+import java.io.PrintStream;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.apex.malhar.lib.function.Function;
+import org.apache.apex.malhar.lib.function.FunctionOperator;
 import org.apache.hadoop.conf.Configuration;
 
+import com.google.common.annotations.VisibleForTesting;
+
 import com.datatorrent.api.Context;
 import com.datatorrent.api.DAG;
 import com.datatorrent.api.StreamingApplication;
 import com.datatorrent.api.annotation.ApplicationAnnotation;
 import com.datatorrent.common.partitioner.StatelessPartitioner;
-import com.datatorrent.lib.io.ConsoleOutputOperator;
 import com.datatorrent.lib.transform.TransformOperator;
 
 @ApplicationAnnotation(name = "TransformExample")
 /**
  * @since 3.7.0
  */
-public class Application implements StreamingApplication
+public class SimpleTransformApplication implements StreamingApplication
 {
+  @SuppressWarnings("serial")
+  public static final Function.MapFunction<Object, Void> CONSOLE_OUTPUT_FN = new Function.MapFunction<Object, Void>()
```

Comment:
I actually thought this can be moved to common package where all examples can use it from.

Reply:
Introduced a class FunctionOperatorUtil into which the above code is moved. This new class is in the same package as the FunctionOperator class.